lib/properties.js

```diff
@@ -148,6 +148,11 @@ class Properties extends EventEmitter {
    * @return {Object}               The destination object
    */
   static merge(destination, source) {
+    // Only merge source if it's an Object. `== null ` covers both null and undefined
+    if (source == null || Object.getPrototypeOf(source) !== Object.prototype) {
+      return destination;
+    }
+
```

Comment:
This should go after the `destination` check so that Properties.merge always returns an Object